library/std/src/panic.rs

```diff
@@ -296,9 +296,12 @@ pub fn get_backtrace_style() -> Option<BacktraceStyle> {
     }
 
     // Setting environment variables for Fuchsia components isn't a standard
-    // or easily supported workflow. For now, display backtraces by default.
+    // or easily supported workflow. For now, display backtraces by default,
+    // and allow turning off completely for compiler tests.
```

Comment:
```suggestion
    // and allow turning off completely.
```

Compiler tests motivated this PR but it's a generally useful feature.